pyproject.toml

```diff
@@ -16,6 +16,8 @@ psycopg-binary = "^3.1.18"
 [tool.poetry.group.dev.dependencies]
 black = "^24.4.0"
 # add test-containers to boot up postgres database for testing in CI/CD
+testcontainers = {version = "^4.4.0", extras = ["postgres"]}
+coverage = "^7.5.0"
```

Comment:
could even move this to another group like `cicd` but it'd only contain Testcontainers and I'm not sure that's worth it. So for now, we'll just install this with group `dev`